tools/StaticAnalysis/Exceptions/BreakingChangeIssues.csv

```diff
@@ -781,3 +781,6 @@
 "Microsoft.Azure.Commands.Compute.dll","Microsoft.Azure.Commands.Compute.SetAzureVMCommand","Set-AzureRmVM","0","2080","The parameter 'Generalized' in parameter set 'GeneralizeIdParameterSetName' for cmdlet 'Set-AzureRmVM' no longer has the attribute 'ValueFromPipelineByPropertyName'.","Add the attribute 'ValueFromPipelineByPropertyName' back to parameter 'Generalized' in parameter set 'GeneralizeIdParameterSetName'."
 "Microsoft.Azure.Commands.Compute.dll","Microsoft.Azure.Commands.Compute.SetAzureVMCommand","Set-AzureRmVM","0","2060","The position of parameter 'Redeploy' has changed for parameter set 'RedeployIdParameterSetName' for cmdlet 'Set-AzureRmVM'.","Revert the position change made to parameter 'Redeploy' for parameter set 'RedeployIdParameterSetName'."
 "Microsoft.Azure.Commands.Compute.dll","Microsoft.Azure.Commands.Compute.SetAzureVMCommand","Set-AzureRmVM","0","2080","The parameter 'Redeploy' in parameter set 'RedeployIdParameterSetName' for cmdlet 'Set-AzureRmVM' no longer has the attribute 'ValueFromPipelineByPropertyName'.","Add the attribute 'ValueFromPipelineByPropertyName' back to parameter 'Redeploy' in parameter set 'RedeployIdParameterSetName'."
+"d:\workspace\powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.CognitiveServices\Microsoft.Azure.Commands.Management.CognitiveServices.dll","Microsoft.Azure.Commands.Management.CognitiveServices.GetAzureCognitiveServicesAccountCommand","Get-AzureRmCognitiveServicesAccount","0","3000","The type of property 'ProvisioningState' of type 'Microsoft.Azure.Commands.Management.CognitiveServices.Models.PSCognitiveServicesAccount' has changed from 'System.Nullable`1[Microsoft.Azure.Management.CognitiveServices.Models.ProvisioningState]' to 'System.String'.","Change the type of property 'ProvisioningState' back to 'System.Nullable`1[Microsoft.Azure.Management.CognitiveServices.Models.ProvisioningState]'."
```

Comment:
not allowed

Reply:
@felixwa breaking changes are not allowed until the breaking change release.  You will need to deprecate this field and adapt to the breaking change in the wire - likely by intrpoducing a new field to your output type, and providing the best available mapping in the existing enum property